dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.httpStat-monitor1.0.jakarta.feature

```diff
@@ -1,10 +1,11 @@
 -include= ~${workspace}/cnf/resources/bnd/feature.props
 symbolicName=io.openliberty.httpStat-monitor1.0.jakarta
 Manifest-Version: 1.0
+IBM-API-Package: io.openliberty.http.monitor.mbean; type="ibm-api"
 IBM-Provision-Capability: \
-  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(|(osgi.identity=io.openliberty.pages-3.1)(osgi.identity=com.ibm.websphere.appserver.servlet-6.0)(osgi.identity=com.ibm.websphere.appserver.servlet-6.1)))", \
+  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(|(osgi.identity=io.openliberty.pages-3.1)(osgi.identity=com.ibm.websphere.appserver.servlet-6.0)(osgi.identity=io.openliberty.servlet.internal-6.0)(osgi.identity=com.ibm.websphere.appserver.servlet-6.1)(osgi.identity=io.openliberty.servlet.internal-6.1)))", \
   osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=com.ibm.websphere.appserver.monitor-1.0))"
 IBM-Install-Policy: when-satisfied
 -bundles=io.openliberty.http.monitor
-kind=noship
-edition=full
\ No newline at end of file
+kind=beta
+edition=core
```

Comment:
Since the feature criteria match GA features, it will be included in the non beta images as well.  You need to add beta guards in the runtime or have a beta feature for this that is only there for the beta until we GA it as an auto feature.